src/Native/pkg/runtime.native.System.IO.Compression/runtime.native.System.IO.Compression.pkgproj

```diff
@@ -9,8 +9,20 @@
   </PropertyGroup>
 
   <ItemGroup>
-  <RuntimeDependency Include="runtime.win10-amd64-aot.runtime.native.System.IO.Compression">
-      <TargetRuntime>win10-amd64-aot</TargetRuntime>
+    <RuntimeDependency Include="runtime.win7-x64.runtime.native.System.IO.Compression">
+      <TargetRuntime>win7-x64</TargetRuntime>
+      <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
+    </RuntimeDependency>
+    <RuntimeDependency Include="runtime.win7-x86.runtime.native.System.IO.Compression">
+      <TargetRuntime>win7-x86</TargetRuntime>
+      <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
+    </RuntimeDependency>
+    <RuntimeDependency Include="runtime.win8-arm.runtime.native.System.IO.Compression">
+      <TargetRuntime>win8-arm</TargetRuntime>
+      <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
+    </RuntimeDependency>
+    <RuntimeDependency Include="runtime.win10-x64-aot.runtime.native.System.IO.Compression">
+      <TargetRuntime>win10-x64-aot</TargetRuntime>
       <Version>4.0.1-$(ExternalExpectedPrerelease)</Version>
```

Comment:
Looks good.  I've confirmed all these IDs match what we have on myget.

On the overall approach (something to consider for Master) we can actually keep the project references around and just have the projects themselves choose a different prerelease value depending on a lookup table.  That will help us manage this in a single central place rather than hacking up the projects.  For details see VersionSuffix and ApplyPreReleaseLabel in https://github.com/dotnet/buildtools/blob/master/src/Microsoft.DotNet.Build.Tasks.Packaging/src/PackageFiles/Packaging.targets.